inc/saiport.h

```diff
@@ -1456,22 +1458,26 @@ typedef sai_status_t (*sai_clear_port_all_stats_fn)(
  *
  * Passed as a parameter into sai_initialize_switch()
  *
+ * @param[in] switch_id Switch id
  * @param[in] count Number of notifications
  * @param[in] data Array of port operational status
  */
 typedef void (*sai_port_state_change_notification_fn)(
+        _In_ sai_object_id_t switch_id,
```

Comment:
why do we need switch id in port notifications ?
each port event contains port object, which already has the switch id encoded in it